src/AspNet.Security.OpenIdConnect.Extensions/OpenIdConnectConstants.cs

```diff
@@ -9,18 +9,41 @@ public static class OpenIdConnectConstants {
         public static class Claims {
             public const string Active = "active";
             public const string Address = "address";
+            public const string AtHash = "at_hash";
+            public const string Birthdate = "birthdate";
             public const string ClientId = "client_id";
             public const string Confidential = "confidential";
+            public const string Email = "email";
```

Comment:
Additionally, Registered JWT claims aren't necessarily ones used by the OpenID Connect spec. `sub` in particular doesn't reference the registered jwt claims, however it's the same claim type. Besides, Maybe not using those constants can help reduce references or dependency on other code?